crc-bundle-info.json.sample

```diff
@@ -7,12 +7,15 @@
   # - 1.1: addition of 'name'
   # - 1.2: addition of 'storage.fileList'
   # - 1.3: remove of 'clusterInfo.kubeadminPasswordFile'
-  "version": "1.3",
+  # - 1.4: addition of 'arch'
+  "version": "1.4",
```

Comment:
1.4 was introduced before this change for `"type": "podman"`, but `crc-bundle-info.json.sample` was not updated accordingly.

Reply:
@cfergeau `type` was already there but value was only `snc` so we just changed the `type` as `podman` for podman branch so there is no key added for info json data.